bt_ddos_shield/utils.py

```diff
@@ -1 +1,4 @@
-Hotkey = str # type of Hotkey
+from typing import TypeAlias
+
+Hotkey: TypeAlias = str
+PublicKey: TypeAlias = str
```

Comment:
:heart:

Reply:
If you develop at head and plan to use 3.12+, which I pretty much think _is_ true, according to https://github.com/bactensor/bt-ddos-shield/blob/c8e6b969a9612692053b6d991f5c1917e201dd7a/pyproject.toml#L8-L9

...then I suggest you use `type` statements from PEP 695 instead:

```suggestion
type Hotkey = str
type PublicKey = str
```

Read more:
- [PEP 695](https://peps.python.org/pep-0695/)
- [the `type` statement](https://docs.python.org/3/reference/simple_stmts.html#type)

Reply:
After this change, such constructs used in my code stopped working:
1. if isinstance(obj, Hotkey):
2. decoded_mapping[Hotkey(hotkey)]

So I'm not changing this, because I think TypeAlias is what I want here.